include/onnxruntime/core/providers/dml/dml_provider_factory.h

```diff
@@ -99,6 +99,18 @@ struct OrtDmlApi {
    * This API gets the D3D12 resource when an OrtValue has been allocated by the DML EP.
    */
   ORT_API2_STATUS(GetD3D12ResourceFromAllocation, _In_ OrtAllocator* provider, _In_ void* dml_resource, _Out_ ID3D12Resource** d3d_resource);
```

Comment:
>GetD3D12ResourceFromAllocation [](http://example.com/codeflow?start=18&length=30)

Should we mark this as deprecated, so new customers dont call into this API anymore?